Xamarin.Forms.Core/ListView.cs

```diff
@@ -217,6 +219,12 @@ public Color SeparatorColor
 			set { SetValue(SeparatorColorProperty, value); }
 		}
 
+		public Color SpinnerColor
```

Comment:
RefreshControlColor?  because it could be something different from a spinner in a specific platform.

Reply:
Hm makes sense. I was struggling with the name. RefreshControlColor seems to cover it.